app/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -440,6 +466,11 @@ private void updateAccountAndRoles(Study study, Set<Roles> callerRoles, Account
             updateRoles(callerRoles, participant, account);
         }
     }
+    
+    private <T> void updatePersistedSet(Set<T> persisted, Set<T> updated) {
+        persisted.removeAll(Sets.difference(persisted, updated));
+        persisted.addAll(updated);
```

Comment:
Would it be easier just to call account.setAccountSubstudies()?

Reply:
This doesn't work. The list implementation being returned by Hibernate is "aware" of its member elements and will update when we update the account. But if you replace this list entirely with something like an ArrayList, this update doesn't happen correctly (SFAICT adding items to the array works, but removing them does not). I did realize yesterday evening that clear() and addAll() might be simpler and work just as well, I'll try it.

Reply:
Yep that works fine, updating.